<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>581</width>
    <height>397</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Pick One</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow {
	
	border-image: url(GUI_Elements/bg image.png);
}
QLabel {
  color: #464d55;
  font-weight: 600;
}
QLabel#heading {
  color: #0f1925;
  font-size: 18px;
  margin-bottom: 10px;
}

QLabel#subheading {
  color: #0f1925;
  font-size: 12px;
  font-weight: normal;
  margin-bottom: 10px;
}
QLineEdit {
  background: transparent;
  color: rgb(86, 86, 86);
  border-bottom: 1px solid rgb(86, 86, 86);
}

QPushButton {
  background-color: #85E3D9;
  color: #0f1925;
  font-weight: 600;
  border-radius: 8px;
  border: 1px solid #85E3D9;
  padding: 5px 15px;
  margin-top: 10px;
  outline: 0px;
}
QPushButton:hover,
QPushButton:focus {
  background-color: #85E3D9;
  border: 3px solid #85E3D9;
}
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>90</y>
      <width>321</width>
      <height>191</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
border-radius: 8px;</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>30</y>
       <width>131</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 16pt &quot;MS Shell Dlg 2&quot;;

color: rgb(86, 86, 86);</string>
     </property>
     <property name="text">
      <string>Who are you?</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>50</y>
       <width>111</width>
       <height>91</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">image: url(GUI_Elements/10.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="adminButton">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>120</y>
       <width>111</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton:hover,
QPushButton:focus {
	
  color: rgb(255, 255, 255);
  background-color: #85E3D9;
  border: 3px solid #85E3D9;
}</string>
     </property>
     <property name="text">
      <string>Administrator</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>50</y>
       <width>111</width>
       <height>91</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">image: url(GUI_Elements/9.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="studentButton">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>120</y>
       <width>111</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton:hover,
QPushButton:focus {
	
  color: rgb(255, 255, 255);
  background-color: #85E3D9;
  border: 3px solid #85E3D9;
}</string>
     </property>
     <property name="text">
      <string>Student</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>-10</y>
      <width>271</width>
      <height>291</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(GUI_Elements/2.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>150</y>
      <width>351</width>
      <height>291</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(GUI_Elements/2.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>581</width>
     <height>18</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="GUI Elements/OptionScreen.qrc"/>
 </resources>
 <connections/>
</ui>
